src/components/Toast.js

```diff
@@ -70,7 +70,7 @@ const Toast = (props) => {
 	};
 
 	return (
-		<div className={className} style={style} {...(props.onClick ? clickProps : {})}>
+		<div aria-live="polite" className={className} role="status" style={style} {...(props.onClick ? clickProps : {})}>
```

Comment:
Can we accept these via the options object? These values could be the default if none is specified in the options.

Reply:
As far as I understand the wai-aria specs, I don't really see the need to make these optional/condigurable.